sharepoint-server-farm-ha/azuredeploy.json

```diff
@@ -0,0 +1,1985 @@
+		},
+		"spVMSize": {
+			"type": "string",
+			"allowedValues": [
+				"Standard_D1",
+				"Standard_DS1",
+				"Standard_D2",
+				"Standard_DS2",
+				"Standard_D3",
+				"Standard_DS3",
+				"Standard_D4",
+				"Standard_DS4",
+				"Standard_D11",
+				"Standard_DS11",
+				"Standard_D12",
+				"Standard_DS12",
+				"Standard_D13",
+				"Standard_DS13",
+				"Standard_D14",
+				"Standard_DS14"
+			],
+			"metadata": {
+				"Description": "The size of the SP VMs Created"
+			},
+			"defaultValue": "Standard_DS2"
+		},
+		"adImagePublisher": {
+			"type": "string",
+			"defaultValue": "MicrosoftWindowsServer",
+			"metadata": {
+				"Description": "Image Publisher"
+			}
+		},
+		"adImageOffer": {
+			"type": "string",
+			"defaultValue": "WindowsServer",
+			"metadata": {
+				"Description": "Image Offer"
+			}
+		},
+		"adImageSKU": {
+			"type": "string",
+			"defaultValue": "2012-R2-Datacenter",
+			"metadata": {
+				"Description": "Image SKU"
+			}
+		},
+		"sqlImagePublisher": {
+			"type": "string",
+			"defaultValue": "MicrosoftSQLServer",
+			"metadata": {
+				"Description": "Image Publisher"
+			}
+		},
+		"sqlImageOffer": {
+			"type": "string",
+			"defaultValue": "SQL2014-WS2012R2",
+			"metadata": {
+				"Description": "Image Offer"
+			}
+		},
+		"sqlImageSKU": {
+			"type": "string",
+			"defaultValue": "Enterprise",
+			"metadata": {
+				"Description": "Image SKU"
+			}
+		},
+		"witnessImagePublisher": {
+			"type": "string",
+			"defaultValue": "MicrosoftWindowsServer",
+			"metadata": {
+				"Description": "Image Publisher"
+			}
+		},
+		"witnessImageOffer": {
+			"type": "string",
+			"defaultValue": "WindowsServer",
+			"metadata": {
+				"Description": "Image Offer"
+			}
+		},
+		"witnessImageSKU": {
+			"type": "string",
+			"defaultValue": "2012-R2-Datacenter",
+			"metadata": {
+				"Description": "Image SKU"
+			}
+		},
+		"spImagePublisher": {
+			"type": "string",
+			"defaultValue": "MicrosoftSharePoint",
+			"metadata": {
+				"Description": "Image Publisher"
+			}
+		},
+		"spImageOffer": {
+			"type": "string",
+			"defaultValue": "MicrosoftSharePointServer",
+			"metadata": {
+				"Description": "Image Offer"
+			}
+		},
+		"spImageSKU": {
+			"type": "string",
+			"defaultValue": "2013",
+			"metadata": {
+				"Description": "Image SKU"
+			}
+		},
+		"domainName":{
+			"type":"string",
+			"metadata": {
+				"Description": "The FQDN of the AD Domain created "
+			},
+			"defaultValue":"sphafarm.com"
+		},
+		"dnsPrefix": {
+			"type":"string",
+			"metadata": {
+				"Description": "The DNS Prefix for the Public IP Address for the Sharepoint Web Application"
+			}
+		},
+		"rdpDNSPrefix": {
+			"type":"string",
+			"metadata": {
+				"Description": "The DNS Prefix for the Public IP Address for RDP"
+			}
+		},
+		"spCentralAdminDNSPrefix": {
+			"type":"string",
+			"metadata": {
+				"Description": "The DNS Prefix for the Public IP Address for the Central Admin site"
+			}
+		},
+		"rdpPort":{
```

Comment:
How many people actually change this? For those people does it make sense to have them edit the template?